### PR TITLE
Update create first snap

### PR DESCRIPTION
**Pull request by popey**
Update first snap creation tutorial to use multipass / bases and better best practices.

### CODE REVIEW

**Comment by degville** [comment]
@popey same for this one:
```
59 ![IMAGE](./hero-artwork.png)
```

**Review by cjp256 — CHANGES_REQUESTED** [review]
I think these updates are important so newcomers like myself don't stumble in the introduction :D

The second commit `Update basic snap usage` removes the hero-artwork.png from basic-snap-usage tutorial.  That commit probably wasn't intentional and should be removed (as it is still referenced in that tutorial).

File: tutorials/packaging/create-your-first-snap/create-your-first-snap.md [hunk 1]
```diff
@@ -78,9 +77,18 @@ sudo snap install --classic snapcraft
 
 positive
 : **NOTE:**
-    * If the `snap` command is not available, install the `snapd` package via APT.
     * The `--classic` switch enables the installation of a snap that uses *classic confinement*.  We discuss snap security confinement in the following section.
 
+Next we will install multipass to manage the Virtual Machines in which snapcraft will build.
```
Comment (cjp256) [inline tutorials/packaging/create-your-first-snap/create-your-first-snap.md]: I don't think this section is required anymore.  It looks like multipass is shipping with the latest stable snapcraft.

File: tutorials/packaging/create-your-first-snap/create-your-first-snap.md [hunk 2]
```diff
@@ -302,13 +291,13 @@ Let's try to execute it:
...
+On traditional Ubuntu you will get output similar to:
 
 ```no-highlight
-The program 'hello' can be found in the following packages:
- * hello
- * hello-traditional
-Try: sudo apt install <selected package>
+Command 'hello' not found, but can be installed with:
+
+sudo apt install hello
```
Comment (cjp256) [inline tutorials/packaging/create-your-first-snap/create-your-first-snap.md]: NIT: trailing whitespace

File: tutorials/packaging/create-your-first-snap/create-your-first-snap.md [hunk 3]
```diff
@@ -553,15 +522,14 @@ built-in help system. You can discover all the options by running:
 Now run:
 
 ```bash
-snapcraft clean gnu-bash -s build
+snapcraft clean gnu-bash
```
Comment (cjp256) [inline tutorials/packaging/create-your-first-snap/create-your-first-snap.md]: Testing this, I did not get the error described here.  Maybe there is a change which eliminates this error and we can simply remove this section (and perhaps find a new example for configflags).  Removing the section will also address the follow-up usage of `snapcraft prime`

**Comment by cjp256** [comment]
I tested these changes locally after dropping the commit which removes hero-artwork.png, and everything appeared to render correctly.

**Comment by cjp256** [comment]
FWIW, the easiest way to get this running locally in my experience, just simply exec:
```
./run serve tutorials
```

**Comment by steverydz** [comment]
Thanks for submitting a PR for this tutorial. We have moved our tutorials to discourse. To contribute going forwards please go to https://discourse.ubuntu.com/t/create-your-first-snap/14006/2

**Pull request closed without merging**